doc/README.librexgen

```diff
@@ -111,9 +167,45 @@ rexgen '[0-2]password[A-C]'
 1passwordC
 2passwordC
 
-Once you are at this point, rexgen is installed.  JtR ./configure now needs
+
+Once rexgen is installed, JtR ./configure now needs
 --enable-rexgen to detect and use the library.
 
+
+KNOWN ISSUES
+
+1.
+The rexgen manpage includes this example:
+
+Examples
+       rexgen index.php?id=[1-5]
+              Would create the results
+              index.php?id=1
+              index.php?id=2
+              index.php?id=3
+              index.php?id=4
+              index.php?id=5
+
+However, to produce the desired output, you have to run
+       rexgen 'index\.php\?id=[1-5]'
+
+(Note the single quotes and the need to escape both '.' and '?'.)
+
+2.
+While some invalod regular expressions are correctly detected and reported,
```

Comment:
You probably meant `invalid`, unless you deliberately invalidated this word. ;-)